symbols/overlay29.yml

```diff
@@ -8500,6 +8500,44 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: GetAiUseItemProbability
+      address:
+        EU: 0x231EAC4
+        NA: 0x231DEB8
+        JP: 0x231F138
+      description: |-
+        Called to get the probability of an item being used or thrown by an AI on the current turn.
+
+        r0: Pointer to either the user if it is an item used by the AI or the target if it is an item thrown by the AI
+        r1: Pointer to item
+        r2: Unknown 2-bit bitvector. Bit 1 influences only the Warp Seed case, while Bit 0 influences various items.
```

Comment:
In the [equivalent function in Rescue Team](https://github.com/pret/pmd-red/blob/master/src/dungeon_ai_item_weight.c#L14), r2 represents the AI's current target int context of deciding the item probability.
* If bit 0 is set, the AI is throwing the item to another Pokémon (friend or foe). If not set, the AI is consuming the item.
* If bit 1 is set, the AI is targeting an ally. If not set, the AI is targeting an enemy.

The description says bit 1 is only for the Warp Seed, which is consistent with Rescue Team and leads me to believe the field works the same in both games.